src/store/modules/shipment/ShipmentState.ts

```diff
@@ -1,5 +1,5 @@
 export default interface ShipmentState {
-  current: object;
+  current: any;
```

Comment:
Why we have changed it to any?

Reply:
Sir, I have used any in place of an object because We need to access further items inside current which was only possible by using any.
Thank you sir for reviewing my code.